config/helm/deployment-args-kubeadm-control-plane-controller-manager.yaml

```diff
@@ -13,4 +13,3 @@ spec:
             - --feature-gates=ClusterTopology=true,KubeadmBootstrapFormatIgnition=true
             - --watch-filter={{ .Values.watchFilter }}
             - --v=0
-          image: '{{.Values.images.domain}}/{{.Values.images.controlplane.name}}:{{.Values.images.controlplane.tag | default .Values.images.tag}}'
```

Comment:
we don't need this line as there is already an `image` section in `kustomize` which we can use for the `image: ` value replacement (see `kustomize.yaml` changes in this PR)